src/inmanta/server/services/databaseservice.py

```diff
@@ -116,7 +116,7 @@ def start_monitor(self) -> None:
     async def get_connection_status(self) -> bool:
         if self._pool is not None and not self._pool._closing and not self._pool._closed:
             try:
-                async with self._pool.acquire(timeout=10):
+                async with self._pool.acquire(timeout=5):
```

Comment:
I'm curious, what's the rationale behind this change ?

Reply:
oh, right not required, thx